mrtt-ui/src/library/separateOutliersFromInliers.js

```diff
@@ -0,0 +1,21 @@
+const separateOutliersFromInliers = ({ inliersList, listContainingOutliers }) => {
+  let outliers = []
+  let inliers = []
+
+  const inliersListToUse = inliersList ?? []
+  const listContainingOutliersToUse = listContainingOutliers ?? []
+
+  listContainingOutliersToUse.forEach((listItem) => {
+    const isInlier = inliersListToUse.find((inlier) => inlier === listItem)
+
+    if (isInlier) {
+      inliers.push(listItem)
+    } else {
+      outliers.push(listItem)
+    }
+  })
+
+  return { outliers, inliers }
+}
+
```

Comment:
prob wont test this, but will test the mapDataForApi function when we solidify our approaches

Reply:
was this function deleted? don't see it in library anymore

Reply:
ya, not necessary. Found a better way